HorizonEditor/src/ContentBrowser/ContentBrowser.cpp

```diff
@@ -144,7 +144,7 @@ void ContentBrowser::OnImGuiRender()
 				{
 					HZN_WARN("item");
 					itemWasSelected = true;
-					selected_file = filenameString;
+					selected_file = path.string();
```

Comment:
I see, this was what was causing the previous error, this should solve the issue.

Reply:
Yeah, now there should not be crash in case of deletion or creation of file or folder based on the selection.